src/components/Map/Map.js

```diff
@@ -45,15 +46,23 @@ const MarkersList = ({ markers }) => {
 // Initial map value, before the user provide their coordinates.
 const balfur = [31.7749837, 35.219797];
 
-function AppMap({ markers, coordinates, setMapPosition }) {
+function AppMap({ markers, coordinates, setMapPosition, setCoordinates }) {
+  const addressInputRef = useRef(); //search input ref
```

Comment:
Please make it neat:
```suggestion
  const addressInputRef = useRef(); // Search Bar ref
```